### PR TITLE
added ge ops and KP details

### PR DESCRIPTION
**Pull request by Ak-sky**
### Description

Added ge ops and KP details

### Release required?

- [x] No release
- [ ] Patch release (`x.x.X`)
- [ ] Minor release (`x.X.x`)
- [ ] Major release (`X.x.x`)

### Run the pipeline

If the CI pipeline doesn't run when you create the PR, the PR requires a user with GitHub collaborators access to run the pipeline.

Run the CI pipeline when the PR is ready for review and you expect tests to pass. Add a comment to the PR with the following text:

```
/run pipeline
```

### Checklist for reviewers

- [ ] If relevant, a test for the change is included or updated with this PR.
- [ ] If relevant, documentation for the change is included or updated with this PR.

### For mergers

- Use a conventional commit message to set the release level. Follow the [guidelines](https://terraform-ibm-modules.github.io/documentation/#/merging.md).
- Include information that users need to know about the PR in the commit message. The commit message becomes part of the GitHub release notes.
- Use the **Squash and merge** option.

### CODE REVIEW

**Review by ocofaigh — CHANGES_REQUESTED** [review]
see comments

File: common-go-assets/common-permanent-resources.yaml [hunk 1]
```diff
@@ -69,3 +69,12 @@ imported_certificate_sm_region: "us-south"
 imported_certificate_intermediate_secret_id: "1a5dba6b-eec7-578d-0973-26994233ab5c"
 imported_certificate_public_secret_id: "029457f1-e9f6-ff13-4afa-ffe7f14d29bc"
 imported_certificate_private_secret_id: "c54fb897-8737-46d3-d8f3-3b59fc32f38d"
+
+# GE Ops Account Id
+ge_ops_account_id: "9f9af00a96104f49b6509aa715f9d6a5" # pragma: allowlist secret
+
+# GE Ops KP Instance and Root Keys
+kp_south: "44f9c10d-99f5-4547-9e9f-2a1c84b5f0a4"
```
[the remark this answers is not in the document]
Comment (Ak-sky) [reply common-go-assets/common-permanent-resources.yaml]: done

File: common-go-assets/common-permanent-resources.yaml [hunk 2]
```diff
@@ -69,3 +69,12 @@ imported_certificate_sm_region: "us-south"
...
+kp_name_south: "perm-KP-base-ocp"
```
[the remark this answers is not in the document]
Comment (Ak-sky) [reply common-go-assets/common-permanent-resources.yaml]: Done

File: common-go-assets/common-permanent-resources.yaml [hunk 3]
```diff
@@ -69,3 +69,12 @@ imported_certificate_sm_region: "us-south"
...
+kp_rg_south: "Default"
+kp_south_root_key_crn: "crn:v1:bluemix:public:kms:us-south:a/9f9af00a96104f49b6509aa715f9d6a5:44f9c10d-99f5-4547-9e9f-2a1c84b5f0a4:key:f6c9f6d0-92f6-437a-b97c-4b617cb3d320"
```
[the remark this answers is not in the document]
Comment (Ak-sky) [reply common-go-assets/common-permanent-resources.yaml]: Done

**Review by ocofaigh — APPROVED** [review]

**Pull request merged**